diff -NurwBd coreutils-8.5.orig/tests/misc/ls-time coreutils-8.5/tests/misc/ls-time
--- coreutils-8.5.orig/tests/misc/ls-time	2010-04-20 19:52:05.000000000 +0000
+++ coreutils-8.5/tests/misc/ls-time	2010-07-04 10:50:25.420327645 +0000
@@ -23,6 +23,8 @@
 
 . $srcdir/test-lib.sh
 
+skip_test_ 'This test fails under btrfs'
+
 # Avoid any possible glitches due to daylight-saving changes near the
 # time stamps used during the test.
 TZ=UTC0
